### PR TITLE
Feature layout first screen

### PR DESCRIPTION
**Pull request by maira-o**
time-into-coins-first-screenshot

Logo image is centralized;
Inactive pause button and play button are positioned;
TouchableOpacity onPress is only available on play button,

### CODE REVIEW

File: src/components/common/Button.js [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
 
 const Button = ({ onPress }) => {
     return (
-      <TouchableOpacity onPress={onPress}>
+      <View style={styles.containerButton}>
         <Image
-          style={styles.buttonPlayStyle}
-          source={require('../../../assets/images/botao-play.png')}
+          style={styles.buttonStyle}
+          source={require('../../../assets/images/botao-pause-inativo.png')}
```
Comment (Morbix) [inline src/components/common/Button.js]: Your commn button should be generic

File: src/components/common/Button.js [hunk 2]
```diff
@@ -1,23 +1,34 @@
...
         />
-      </TouchableOpacity>
+
+        <TouchableOpacity onPress={onPress}>
+          <Image
+            style={styles.buttonStyle}
+            source={require('../../../assets/images/botao-play.png')}
```
Comment (Morbix) [inline src/components/common/Button.js]: Your common button should be generic

Comment (maira-o) [reply src/components/common/Button.js]: Ok, I'll modify this!

**Comment by maira-o** [comment]
@Morbix I changed the specific button to generic and rename the form. Should be like this?

File: App.js [hunk 3]
```diff
@@ -1,28 +1,42 @@
 import React from 'react';
-import { View, Image, ScrollView } from 'react-native';
+import { View, Image, ScrollView, KeyboardAvoidingView } from 'react-native';
 import Header from './src/components/common/Header';
-import InputForm from './src/components/InputForm';
+import MeetingInfoForm from './src/components/MeetingInfoForm';
```
Comment (Morbix) [inline App.js:4]: 👍

**Review by Morbix — APPROVED** [review]

**Pull request merged**